scripts/Package_BGA/bga.py

```diff
@@ -18,8 +18,9 @@ def bga(args):
 
     pitch = args["pitch"]
     padDiameter = args["pad_diameter"]
-    pasteRatio = args["paste_ratio"]
-    pasteDiameter = args["paste_diameter"]
+    maskMargin = args["mask_margin"]
+    pasteMargin = args["paste_margin"]
+    pasteRatio = args["paste_ratio"] # '10.0' means paste ratio is 10% of pad size
```

Comment:
I think the comment is no longer correct

Reply:
I deleted that comment.